<launch>
  <node pkg="alego" type="IP" name="IP" />

  <node pkg="alego" type="LO" name="LO" output="log">
    <!-- <remap from="/segmented_cloud" to="/segmented_cloud" />
    <remap from="/seg_info" to="/segmented_cloud_info" />
    <remap from="/outlier" to="/outlier_cloud" /> -->
  </node>
  <node pkg="alego" type="LM" name="LM" output="log">
    <!-- <remap from="/outlier" to="/outlier_cloud" /> -->
  </node>
</launch>